crates/swc_ecma_minifier/src/pass/mangle_names/private_name.rs

```diff
@@ -5,11 +5,15 @@ use swc_ecma_ast::*;
 use swc_ecma_utils::{ident::IdentLike, Id};
 use swc_ecma_visit::{as_folder, noop_visit_mut_type, Fold, VisitMut, VisitMutWith};
 
-pub fn private_name_mangler(keep_private_props: bool) -> impl Fold + VisitMut {
+pub fn private_name_mangler(
+    keep_private_props: bool,
+    reserved: Vec<String>,
```

Comment:
`JsWorrd` would be better.

Reply:
resolved in c12701c9208b636d660ef23458d9aaf87706d2bc